index.html

```diff
@@ -1,50 +1,274 @@
+                    <ul class="tonic-title">
+                    <li class="tonic-item">CONOPY <img src="images/Counter.png" alt=""></li>
+                    <li class="tonic-item">Back End Dev <img src="images/Counter.png" alt=""></li>
+                    <li class="tonic-item">2015 <img src="images/Counter.png" alt=""></li>
+                    </ul>
+
+                    
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                
+                    <div>
+                        
+                        <div>
+                            <ul class="skills">
+                                <li  class="html-link">HTML</li>  
+                                <li  class="html-link">CSS</li>
+                                <li  class="html-link">javascript</li>
+                            </ul>
+                        </div>
         
-        <section class="sub-container">
+                    </div>
+
+                    <div>
+                        <button class="project-button">See Project</button>
+                    </div>
+                </section>
+            </section>
+
+            <section class="tonic">
+                <section class="tonic-container">
+                    
+                    <div class="section3"></div>
+
+                    <h2>Tonic</h2>
+                
+                    <ul class="tonic-title">
+                    <li class="tonic-item">CONOPY <img src="images/Counter.png" alt=""></li>
+                    <li class="tonic-item">Back End Dev <img src="images/Counter.png" alt=""></li>
+                    <li class="tonic-item">2015 <img src="images/Counter.png" alt=""></li>
+                    </ul>
+
+                    
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                
+                    <div>
+
+                        <div>
+                            <ul class="skills">
+                                <li  class="html-link">HTML</li>  
+                                <li  class="html-link">CSS</li>
+                                <li  class="html-link">javascript</li>
+                            </ul>
+                        </div>
         
-            <h1 class="heading">I’m Mariane
-            Glad to see you!</h1>
+                    </div>
 
-            <p class="paragraph">I’m a software developer! 
-            I can help you build a product , 
-            feature or website Look through some of my work and experience! 
-            If you like what you see and have a project you need coded, 
-            don’t hestiate to contact me.</p>
+                    <div>
+                        <button class="project-button">See Project</button>
+                    </div>
+                </section>
+            </section>
 
-            <div class="socials">   
+            <section class="tonic">
```

Comment:
- [ ] As mentioned by the previous reviewers, I see that you have each project in a section element. It would have been better to have `section` container that wraps all the six project. Based on the requirement for this project, you are expected to use `display:grid` to align the content of the `section` container. Kindly implement this as it is a core requirement for this project. e.g 

```
<section class="projects-container">
    <div>Project one</div>    
    <div>Project two</div>    
    <dinv>Project three</div>    
    <div>Project four</div>    
    <div>Project five</div>
    And so on
  </section>

.projects-container {
display: grid;
other properties
}
```